### ISSUE TITLE
(add_MARBL): prepare talk for webinar

### ISSUE THREAD

**Issue by mnlevy1981** [mnlevy1981/MOM6#30]
Rough idea for an outline -- break down each stage into list of tasks with visual for "done", "in progress", "not yet started"

1. Infrastructure

   * Build MARBL with MOM [done, but need better way to incorporate `-D_USE_MARBL_TRACERS`
   * Create compsets
   * Generate `marbl_in`
   * Generate MARBL contributions to diagnostic table [new stream file]

1. Init

   * make sure all tasks have copy of `marbl_in`
   * call `marbl_instance%init()`
   * Set up memory for diagnostics

1. Compute surface flux

   * Set all forcings, surface tracer values, and saved state
   * call `marbl_instance%surface_flux_compute()`
   * Update saved state
   * If writing restart file, add saved state
   * accumulate diagnostics
   * apply surface fluxes appropriately
   * [infrastructure] call multiple columns at once (POP avoids packing in land cells)

1. Compute interior tendency

   * Same as above, but sub tendency for flux and call single column at a time
   * [infrastructure] don't call for halo cells or land cells

1. Shutdown model

   * Write any remaining log messages to `ocn.log`
   * clean up memory

1. Other

   * Cost of adding MARBL to the model?

**Comment by mnlevy1981** [comment]
Maybe the talk can use lots of flow charts that are color-coordinated to show what is done, what is in progress, and what is still missing? (solid connector for done, dashed for in progress, dotted for "future work"?)
